helm/cluster/README.md

```diff
@@ -781,6 +781,7 @@ Provider-specific properties that can be set by cluster-$provider chart in order
 | `providerIntegration.kubeadmConfig.postKubeadmCommands[*]` |**None**|**Type:** `string`<br/>|
 | `providerIntegration.kubeadmConfig.preKubeadmCommands` | **Pre-kubeadm commands** - Extra commands to run before kubeadm runs.|**Type:** `array`<br/>|
 | `providerIntegration.kubeadmConfig.preKubeadmCommands[*]` |**None**|**Type:** `string`<br/>|
+| `providerIntegration.kubeadmConfig.users` | **Users** - If enabled, the giantswarm user will be created on all nodes.|**Type:** `boolean`<br/>**Default:** `false`|
```

Comment:
the only thing i'm not sure about is the location of the newly-added toggle in the values - is this satisfactory for Team :turtle:?

Reply:
I think that the location is fine, it's alongside other kubeadm things.

Other kubeadm properties here are I think mostly aligned with what is set in kubeadm (similar structures), so maybe check the other comment here for naming boolean vs list https://github.com/giantswarm/cluster/pull/296#discussion_r1716578448